qbittorrent/client.py

```diff
@@ -615,9 +615,8 @@ def set_file_priority(self, infohash, file_id, priority):
         :param infohash: INFO HASH of torrent.
         :param file_id: ID of the file to set priority.
         :param priority: Priority level of the file.
-        :note priority 4 is no priority set
         """
-        if priority not in [0, 1, 2, 4, 7]:
+        if priority not in [0, 1, 6, 7]:
             raise ValueError("Invalid priority, refer WEB-UI docs for info.")
```

Comment:
Should we keep 2, 4 for backwards compatibility?

Reply:
I'd say yes, it's easier than versioning the scripts

Reply:
Maybe with some comments as to which priorities are relevant to which qbittorrent version

Reply:
where from priority 4? In github docs wrote
4.1+ 0, 1, 6, 7
3.2.0-4.0.4 also 0, 1, 6, 7
3.1.x 0, 1, 2, 7